Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/common/DateFormattingTest.kt

```diff
@@ -29,12 +29,12 @@ class DateFormattingTest : BaseTest() {
 
     @Test
     fun `format date time`() {
-        val tz = DateTimeZone.forID("Europe/Berlin")
+        val tz = ZoneId.of("Europe/Berlin")
         "2021-08-20T10:03:12Z".formatDateTime(tz) shouldBe "2021-08-20 12:03 UTC +02"
         "2021-08-20T12:03:12+02".formatDateTime(tz) shouldBe "2021-08-20 12:03 UTC +02"
-        "2021-08-20T12:03:12+0200".formatDateTime(tz) shouldBe "2021-08-20 12:03 UTC +02"
```

Comment:
As far as I remember, is was requested to have this specific output with UTC in it

Reply:
yes, I had a look at the specs, the time zone in this example does not parse in java.

Reply:
see https://github.com/corona-warn-app/cwa-app-android/pull/5593/commits/8d5855a330ab7d43a86fd32f3abe04a2be6cedbd